### PR TITLE
Create ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by y509015

Ticket: [MTM-58456](https://cumulocity.atlassian.net/browse/MTM-58456)
Original PR: [5930](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5930)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Default value is displayed on edit mode of asset property in sub-assets page even when user clears the value (#5930)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Asset property default value displayed in sub-asset edit mode after clearing the value
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58456
+version: 1020.0.0
+---
+Default value is displayed on edit mode of asset property in sub-assets page even when user clears the value (#5930)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT application, when editing an asset property value in the sub-assets page, the default value was still displayed even after the user had cleared the value. This issue has now been resolved. With this fix, when a user clears an asset property value in the sub-asset edit mode, the field will be empty as expected without retaining the default value. This change improves the usability and data integrity when modifying asset properties within sub-assets.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58456
+version: 1020.0.0
+---
+When editing an asset property value in the sub-assets page, the default value was still displayed even after the user had cleared the value. This issue has now been resolved. With this fix, when a user clears an asset property value in the sub-asset page and saves the changes, the field will be empty as expected without retaining the default value.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md:17]:
```suggestion
When editing an asset property value in the subassets view, the default value was still displayed even after the user had cleared the value. This issue has now been resolved. With this fix, when a user clears an asset property value in the subassets view and saves the changes, the field will be empty as expected without retaining the default value.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Asset property default value displayed in sub-asset edit mode after clearing the value
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md:3]:
```suggestion
title: Asset property field in subassets view now empty after clearing the value
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes Please review and approve

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md (hunk 4 above)
Comment (rahultsag) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md]: Can we replace the title with "Asset property field in the subassets view now remains empty after the existing or default value is cleared."

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Asset property field in subassets view now empty after clearing the value
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-default-value-is-displayed-on-edit-mode-of-asset-property-in.md:3]:
```suggestion
title: Asset property field in the subassets view now remains empty after the existing or default value is cleared
```

**Review by janhommes — APPROVED** [review]

**Pull request merged**